source/saml/tabs/add-app.rst

```diff
@@ -0,0 +1,25 @@
+Add a new SAML Application on GLPI
+----------------------------------
+
+First of all, you need to add a SAML application on GLPI because we need to report some information on GLPI <-> Entra.
+
+* In **Setup** > **SAML Providers**, click on **+ Add**
+* Give a **name** to your application
+* Click on **is active**
+
+* Click on **Save**
+
+.. image:: images/add-app-glpi.png
+    :alt: Add GLPI app
+    :scale: 42%
+
+* In **Transit** tab, select :
+
+  * Compress requests
+  * Compress respoonses

```

Comment:
Typo: Responses

Reply:
Thanks for your clarification, the changes have been made, I'll push it all through as soon as I've finished the process for Google.